share/runtime-install.tmpl

```diff
@@ -91,14 +86,14 @@ installpkg device-mapper-persistent-data
 installpkg xfsdump
 
 ## needed for LUKS escrow
-installpkg python-volume_key volume_key
-installpkg python-nss
+installpkg volume_key
+installpkg nss-tools
```

Comment:
Why do we need these two? For the kickstart tests? Because other than that libblockdev should correctly require what it needs for LUKS escrow.